### DISCUSSION TITLE
Can't modify CustomWidgets.js template in cookie cutter(-2) app with component option

### DISCUSSION THREAD

**Discussion by moon-chyldren** [Kitware/trame#621] [General] [answered]
Hi! I just created a new trame cookie cutter app using option "2_App with Components" option. 
I attempted to modify the following two lines from the template in the generated "vue-components/CustomWidget.js" file to change the text that appears inside the buttons to 

```
<div>
<v-btn @click="triggerClick" :style="btnClick">Button click</v-btn>
<v-btn @click="triggerClick" :style="btnClick">Button change</v-btn>
</div>
```

and noticed that when I run the app in the browser the text that appears on the buttons still displays the original "CUSTOM CLICK" and "CUSTOM CHANGE" text and NOT the "BUTTON CLICK" and "BUTTON CHANGE" that I just changed it to. 

The text doesn't update even after trying all of the following:

- stop and run the app again
- Hold down `ctrl + F5` in chrome to clear the cache
- Run `npm run build` again inside the vue-components directory 
- (even tried `npm i` to reinstall node_modules` after deleting package-lock.json)
- Going back one directory to the vue-components parent directory and running `pip install -e .` once again
- Running `npm run dev`

I can make other changes to the application from inside of app/core.py and those changes will take but any change I make inside a file ending in *.js has no effect even after trying all of the above. 

If you would like I could provide you with a list of the options I selected when made the cookie cutter and which commands I ran but those commands are all listed inside the autogenerated README.rst file. 
Thanks in advance.

**Comment by jourdain** [comment, marked as answer]
Here is what works for me

```
python3.10 -m venv .venv
source .venv/bin/activate
pip install -e .
cd vue-components
npm i
npm run build
cd ..
test-cookie --server
```

Then in another terminal

```
cd vue-components
# ... edit js
npm run build
# reload browser
```

**Reply by moon-chyldren** [reply]
This works for me as well. Thank you!
